File: Scripts/server/regular-expressions/java/RegularExpressions.java
public class RegularExpressions {

 private static volatile int r = 1;

 public static int executeTask(int i) {

  String str = "this is a matching string" + i;
  boolean ma = (str.matches(".*string")); 
  String strR = str.replaceAll("string", "istyfied"); {}
  return i;
 }

 public static void main(String[] args) {

  for (int i = 0; i < 10000000; ++i) {
  	r = executeTask(i);
  }
 }

}

File: Scripts/server/regular-expressions/javascript/regular-expressions.js
var r = 1;

function executeTask(i) {
    	var str = "this is a matching string" + i; 
     	if( str.match(/string/g)) {}
     	if(str.replace("string", "istyfied")) {}
	return i;
}

for (var i = 0; i < 10000000; ++i) {
	r = executeTask(i);
}
